priv/repo/migrations/20180813212649_create_attendees.exs

```diff
@@ -0,0 +1,16 @@
+defmodule CbusElixir.Repo.Migrations.CreateAttendees do
+  use Ecto.Migration
+
+  def change do
+    create table(:attendees) do
+      add(:name, :string)
+      add(:email, :string)
```

Comment:
This shouldn't be nullable

Reply:
I will fix both of these to be
null: false